src/config/serverConfig.ts

```diff
@@ -5,7 +5,33 @@ export const serverConfig: ServerConfig = {
   profiles: {
     Measure: {
       service: new MeasureService(),
-      versions: [constants.VERSIONS['4_0_1']]
+      versions: [constants.VERSIONS['4_0_1']],
+      operation: [
+        {
+          name: 'package',
+          route: '/$package',
+          method: 'GET',
+          reference: 'https://build.fhir.org/ig/HL7/cqf-measures/OperationDefinition-Measure-package.html'
```

Comment:
When I go to the URL you've linked here, it says:

> The official URL for this operation definition is:

> http://hl7.org/fhir/us/cqfmeasures/OperationDefinition/Measure-package

So I'm going to go with that instead of including the "STU3" bit if that works!